src/firmware/boards/legacy_dongle_stm32f4/BUILD

```diff
@@ -38,13 +37,11 @@ cc_binary(
         # https://docs.bazel.build/versions/master/be/c-cpp.html#cc_binary.includes
         "-isystem firmware/boards/shared/legacy_freertos/include",
         "-isystem firmware/boards/shared/legacy_stm32lib/include",
+        "-isystem firmware/boards/shared/legacy_cdcacm/include",
```

Comment:
since the cdcacm never panned out for the dongle we should remove this?

Reply:
I'm inclined to leave it in so that all the shared libs work in case we need it in the future.